notebooks/influence_wine.ipynb

```diff
@@ -767,6 +775,236 @@
+      "text/plain": [
+       "<Figure size 1600x800 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "plt.plot(mean_ekfac_train_influences, color=\"red\")\n",
```

Comment:
My suggestion would be to use a scatter plot instead, so one could see the correlation immediately.